src/Dialect/ONNX/DialectBuilder.cpp

```diff
@@ -116,6 +116,12 @@ Value OnnxBuilder::constantInt64(const ArrayRef<int64_t> intVals) const {
   return constant(denseAttr);
 }
 
+Value OnnxBuilder::constantFloat32(const ArrayRef<float> floatVals) const {
+  auto shape = RankedTensorType::get({(int64_t)floatVals.size()}, b().getF32Type());
```

Comment:
A small nit: we avoid using `(dtype)` for type cast but `static_cast<int64_t>(value)` instead.